amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-xml-maven.adoc

```diff
@@ -46,24 +41,31 @@ This example shows how the namespace statements are placed in the mule XML block
 
 == Add a POM File Dependency
 
-The `pom.xml` Maven file generated by Anypoint Studio contains dependencies for a Mule app. When you code a Mule app manually, include this XML snippet in your `pom.xml` file to enable access to this connector. The Mule runtime engine translates the `RELEASE` dependency keyword to the current version.
+The Apache Maven `pom.xml` file generated by Anypoint Studio contains dependencies for a Mule app. If you code a Mule app manually, include this XML snippet in your `pom.xml` file to enable access to this connector:
 
 [source,xml,linenums]
 ----
 <dependency>
   <groupId>com.mulesoft.connectors</groupId>
   <artifactId>mule-amazon-s3-connector</artifactId>
-  <version>RELEASE</version>
+  <version>x.x.x</version>
   <classifier>mule-plugin</classifier>
 </dependency>
 ----
 
-To obtain the most up-to-date `pom.xml` file information, access the connector in https://www.mulesoft.com/exchange/[Anypoint Exchange] and click *Dependency Snippets*.
+To obtain the most up-to-date `pom.xml` file information:
```

Comment:
Don't forget to add this text from the template above this paragraph:

Replace `x.x.x` with the version that corresponds to the connector you are using.